### ISSUE TITLE
Can we use ensembldb to convert transcriptomic bam to genomic bam?

### ISSUE THREAD

**Issue by smk5g5** [jotsetung/ensembldb#152]
Hi,

I have a bam file (nanopore transcripts) aligned to cdna and I need to convert it to genomic coordinates. Is there a way to use ensembldb to do that? Kindly let me know.

Thanks

**Comment by jorainer** [comment]
no, sorry, `ensembldb` can not do that. maybe you can do that with the *bamtools* or the Bioconductor `Rsamtools` package?
